### PR TITLE
ref(hybrid-cloud): Mark stable a bunch of control tests that pass while stable

### PR DESCRIPTION
**Pull request by mikejihbe**

### CODE REVIEW

File: tests/sentry_plugins/test_client.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def test_patch(self):
         assert resp.status_code == 200
 
 
-@control_silo_test
+@control_silo_test(stable=True)
```
Comment (markstory) [inline tests/sentry_plugins/test_client.py:54]: Does this need to be control silo? Project plugins are region silo.

File: tests/snuba/sessions/test_sessions.py [hunk 2]
```diff
@@ -1548,7 +1548,7 @@ def test_sesions_per_project_multiple_projects(self):
             assert set(actual) == {(p1.id, 4), (p2.id, 2)}
 
 
-@control_silo_test
+@control_silo_test(stable=True)
```
Comment (markstory) [inline tests/snuba/sessions/test_sessions.py:1551]: session health data is region silo though.

File: tests/sentry/api/serializers/test_organization.py [hunk 3]
```diff
@@ -204,6 +204,7 @@ def test_disable_last_deploys_killswitch(self):
         options.set("api.organization.disable-last-deploys", opt_val)
 
 
+@control_silo_test
```
Comment (corps) [inline tests/sentry/api/serializers/test_organization.py:207]: I believe this is actually a region test by the end.

File: tests/sentry/api/serializers/test_tagvalue.py [hunk 4]
```diff
@@ -3,8 +3,10 @@
 from sentry.api.serializers import UserTagValueSerializer, serialize
 from sentry.tagstore.types import TagValue
 from sentry.testutils import TestCase
+from sentry.testutils.silo import control_silo_test
 
 
+@control_silo_test
```
Comment (corps) [inline tests/sentry/api/serializers/test_tagvalue.py:9]: I believe these two tests are also region silo tests in practice.

File: tests/sentry/web/frontend/test_error_500.py [hunk 5]
```diff
@@ -2,9 +2,11 @@
 from django.urls import reverse
 
 from sentry.testutils import TestCase
+from sentry.testutils.silo import control_silo_test
 
 
 @override_settings(ROOT_URLCONF="sentry.conf.urls")
+@control_silo_test(stable=True)
```
Comment (corps) [inline tests/sentry/web/frontend/test_error_500.py:9]: This actually importaly should be an all_silo_test so we validate this logic is silo independent.

**Review by corps — APPROVED** [review]

**Review by vbro — APPROVED** [review]

**Pull request merged**